backend/src/main/java/gov/cdc/usds/simplereport/service/idverification/LiveExperianService.java

```diff
@@ -89,10 +90,14 @@ private String fetchToken() {
 
     final JsonNodeFactory factory = JsonNodeFactory.instance;
     ObjectNode requestBody = factory.objectNode();
-    requestBody.put("username", _experianProperties.getCrosscoreUsername());
-    requestBody.put("password", _experianProperties.getCrosscorePassword());
-    requestBody.put("client_id", _experianProperties.getClientId());
-    requestBody.put("client_secret", _experianProperties.getClientSecret());
+    String crosscoreUsername = _experianProperties.getCrosscoreUsername();
```

Comment:
⚠️ Specified the values were a `String` -- I think we were running into a similar issue like this one:
https://community.sonarsource.com/t/s1874-fp-deprecated-code-should-not-be-used-java-s1874-in-objectnode/99121/14